File: ruby/q01.rb
i = 11
while true
  if i.to_s == i.to_s.reverse &&
     i.to_s(2) == i.to_s(2).reverse &&
     i.to_s(8) == i.to_s(8).reverse
     puts i.to_s
     puts i.to_s(2)
     puts i.to_s(8)
     break
  end
  i += 1
end

File: js/q01.js
String.prototype.reverse = function () {
    return this.split('').reverse().join('')
}

let i = 11

while (true) {
    if ( (i.toString() == i.toString().reverse()) &&
         (i.toString(2) == i.toString(2).reverse()) &&
         (i.toString(8) == i.toString(8).reverse())
    ) {
        console.log(i.toString(), i.toString(2), i.toString(8))
        break;
    }
    i++
}